src/comps/footer.tsx

```diff
@@ -56,6 +79,35 @@ export default function Footer({ supportLink }: { supportLink?: string }) {
           >
             Github
           </a>
+          <a
+            key="pot"
+            href="https://mempool.space/address/bc1prcs82tvrz70jk8u79uekwdfjhd0qhs2mva6e526arycu7fu25zsqhyztuy"
```

Comment:
This will change once we do another key rotation. Can you read the most recent signer aggregate key from the SC?

Reply:
Will look into this

Reply:
Great idea simple and clean done!